docs/protocol.md

```diff
@@ -163,6 +163,16 @@ Without any mitigation strategy, malicious but protocol adherent nodes can creat
 
 Sidetree protocol defines the following mechanisms to enable scaling, while preventing DDoS attacks:
 
+#### Rate limiting
+   
+   To prevent spam attack causing transaction and operation stores to grow at an unhealthy rate, rate limiting is put in place. 
+
+   The current implementation caps the number of operations allowed to be observed and processed to 1000 a second, and the cap for transactions is set to 300 per transaction time. The selection logic when the caps are exceeded is the following:
+   
+   higher fee per transaction comes first, if transaction fee is the same, lower transaction number comes first.
+   
+   By picking the transactions with higher transaction fee, it encourages batching, while allowing small transactions to have the opportunity to also be included if they are willing to pay a premium of a couple USD. Alternatives to this approach are highest fee per operation and first comes first serve, but they don't encourage batching and discourage writing near the end of a transaction time.
```

Comment:
Suggestion to change the " ... pay a premium of a couple of USD." to something like "... pay a higher transaction fee."